presto-spi/src/main/java/com/facebook/presto/spi/block/PageBuilderStatus.java

```diff
@@ -13,37 +13,27 @@
  */
 package com.facebook.presto.spi.block;
 
-import static com.facebook.presto.spi.block.BlockBuilderStatus.DEFAULT_MAX_BLOCK_SIZE_IN_BYTES;
-
 public class PageBuilderStatus
 {
-    public static final int DEFAULT_MAX_PAGE_SIZE_IN_BYTES = 1024 * 1024;
+    public static final int DEFAULT_MAX_PAGE_SIZE_IN_BYTES = 512 * 1024;
```

Comment:
This is required otherwise test `TestMemoryPools.testBlockingOnUserMemory` will fail. Debugging shows one driver successfully finished (while all other are blocked as expected)

Reply:
Can you elaborate on this?